### PR TITLE
Containers Compaction: pgBouncer pgBadger

### PR DESCRIPTION
**Pull request by jmckulk**
In an effort to remove the crunchy-pg-base image the crunchy-pgbouncer and
crunchy-pgbadger iamges are now built based on the crunchy-base image. Since
neither pgBouncer or pgBadger depend on having PostgreSQL installed, using the
crunchy-pg-base image was making the images larger than they needed to be. This
also removes two more dependencies on the crunchy-pg-base image.

The scripts for crunchy-pgbouncer and crunchy-pgbadger have been updated with
the new `/opt/crunchy` (previously `/opt/cpm`) path when adding binary and
config files. The dockerfiles for these two images now include steps to setup
the required repository files and packager settings based on the baseos. These
steps were previously done in the crunchy-pg-base image.

The pgBouncer dockerfile was modified to not installed the EPEL version of
pgBouncer.

**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change, as applicable?
 - [ ] Have you tested your changes on all related environments with successful results, as applicable?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**

**What is the new behavior (if this is a feature change)?**

**Other information**:

### CODE REVIEW

File: build/pgbadger/Dockerfile [hunk 1]
```diff
@@ -7,14 +7,16 @@ WORKDIR /go/src/github.com/crunchydata/crunchy-containers
...
+FROM ${PREFIX}/crunchy-base:${BASEOS}-${PG_FULL}-${BASEVER}
 
 # For RHEL8 all arguments used in main code has to be specified after FROM
 ARG BASEOS
 ARG DFSET
 ARG PACKAGER
 
 ARG PG_MAJOR
+# Needed due to lack of environment substitution trick on ADD
+ARG PG_LBL
```
Comment (jkatz) [inline build/pgbadger/Dockerfile:19]: Why is this needed? I don't see it being used.

File: build/pgbouncer/Dockerfile [hunk 2]
```diff
@@ -2,13 +2,15 @@ ARG BASEOS
...
-FROM ${PREFIX}/crunchy-pg-base:${BASEOS}-${PG_FULL}-${BASEVER}
+FROM ${PREFIX}/crunchy-base:${BASEOS}-${PG_FULL}-${BASEVER}
 
 # For RHEL8 all arguments used in main code has to be specified after FROM
 ARG DFSET
 ARG PACKAGER
 
 ARG PG_MAJOR
+# Needed due to lack of environment substitution trick on ADD
+ARG PG_LBL
```
Comment (jkatz) [inline build/pgbouncer/Dockerfile:13]: Why is this needed? I don't see it being used.

File: build/pgbadger/Dockerfile (hunk 1 above)
Comment (jmckulk) [reply build/pgbadger/Dockerfile:19]: Its being used to add the `crunchypg${PG_LBL}.repo` files here: https://github.com/CrunchyData/crunchy-containers/pull/1303/files/e37f4073d501710b7aa5338271223a204b58a565#diff-c56704700e72dcb6e7282ad4b8f427230f091b2a1f0c96bbd5b8a5402aea6a51R35

Comment (jkatz) [reply build/pgbadger/Dockerfile:19]: Apparently I have an inability to read. Thanks.

File: build/pgbouncer/Dockerfile (hunk 2 above)
Comment (jmckulk) [reply build/pgbouncer/Dockerfile:13]: same as above

**Review by jkatz — APPROVED** [review]
